src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -67,9 +67,25 @@
         - name: --windows-admin-username
           type: string
           short-summary: User account to create on windows node VMs.
+          long-summary: |-
+            Rules for windows-admin-username:
+                - restriction: Cannot end in "."
+                - Disallowed values: "administrator", "admin", "user", "user1", "test", "user2", "test1", "user3", "admin1", "1", "123", "a", "actuser", "adm", "admin2", "aspnet", "backup", "console", "david", "guest", "john", "owner", "root", "server", "sql", "support", "support_388945a0", "sys", "test2", "test3", "user4", "user5".
+                - Minimum-length: 1 character
+                - Max-length: 20 characters
         - name: --windows-admin-password
           type: string
           short-summary: User account password to use on windows node VMs.
+          long-summary: |-
+            Rules for windows-admin-password:
+                - Minimum-length: 8 characters
+                - Max-length: 123 characters
+                - Complexity requirements: 3 out of 4 conditions below need to be fulfilled
+                  * Has lower characters
+                  * Has upper characters
+                  * Has a digit
+                  * Has a special character (Regex match [\\W_])
+                - Disallowed values:  "abc@123", "P@$$w0rd", "P@ssw0rd", "P@ssword123", "Pa$$word", "pass@word1", "Password!", "Password1", "Password22", "iloveyou!"
```

Comment:
same here, link a doc if we have.

Reply:
seems like our backend service will do the validation?

Reply:
This is to unblock customers to use `-` in the username. Currently AKS relies on CRP to do the validation. AKS will add the validation soon.